src/components/navigation/styled-heading.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
```

Comment:
Do you think we should have a centralized place for all reusable styled components? Is that what you were trying to accomplish with the styled-components folder? If so, this should probably go in that folder somewhere

Reply:
Yeah, that was my idea for the styled-components folder. I just pushed a change that moved this file into the styled-components folder.

Reply:
Good Idea! Thanks for doing this.